### PR TITLE
ListItem v1 (Android & iOS)

### PR DESCRIPTION
**Pull request by martinezguillaume**

### CODE REVIEW

File: src/divider/Divider.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
   style: ViewPropTypes.style,
 };
 
 styles = StyleSheet.create({
   container: {
-    height: 1,
-    backgroundColor: colors.grey5,
+    ...Platform.select({
+      ios: {
+        height: 0.5,
```
Comment (ocarreterom) [inline src/divider/Divider.js]: Should not the `height` be `StyleSheet.hairlineWidth`?
https://facebook.github.io/react-native/docs/stylesheet.html#hairlinewidth

Comment (ocarreterom) [inline src/divider/Divider.js]: Should not the `height` be `StyleSheet.hairlineWidth`?
https://facebook.github.io/react-native/docs/stylesheet.html#hairlinewidth

Comment (martinezguillaume) [reply src/divider/Divider.js]: Good point ! I'll use it !

**Comment by Monte9** [comment]
here we goo.. 🤞

**Pull request merged**